workflows/reference_files/2.4/config/default.yaml

```diff
@@ -138,9 +138,6 @@ tools:
     gsutil: 
         conda_env: "envs/gsutil-4.53.yaml"
         version: "4.53"
-    bedtools:
```

Comment:
I think this bedtools got accidentally dropped, but otherwise looks good

Reply:
It appeared to be duplicated. There's an entry for bedtools above.

Reply:
I see it, thanks! So everything was pushed in this PR and this is the working version?